src/main/java/io/anserini/index/IndexUtils.java

```diff
@@ -57,11 +60,18 @@
     @Option(name = "-dumpTransformedDoc", metaVar = "docid", usage = "dumps transformed document (if stored in the index)")
     String transformedDoc;
 
+    @Option(name = "-dumpSentences", metaVar = "docid", usage = "splits the fetched document into sentences (if stored in the index)")
+    String sentDoc;
+
     @Option(name = "-convertDocidToLuceneDocid", metaVar = "docid", usage = "converts a collection lookupDocid to a Lucene internal lookupDocid")
     String lookupDocid;
 
     @Option(name = "-convertLuceneDocidToDocid", metaVar = "docid", usage = "converts to a Lucene internal lookupDocid to a collection lookupDocid ")
     int lookupLuceneDocid;
+
+    //ToDo: Generalize to any transformer
```

Comment:
Also, remove this for now... since the generators aren't actually going to work and we'll need to refactor the APIs.